### PR TITLE
TFS-65 added office opening date in response

### PR DESCRIPTION
**Pull request by faheem205**
## Description

Describe the changes made and why they were made.
Added Office opening Date in editStaff API response. This will be used on front end to restrict Employee Joiningg Date

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/organisation/staff/api/StaffApiResource.java [hunk 1]
```diff
@@ -143,7 +144,8 @@ public String retrieveOne(@PathParam("staffId") @Parameter(description = "staffI
         final ApiRequestJsonSerializationSettings settings = apiRequestParameterHelper.process(uriInfo.getQueryParameters());
         StaffData staff = readPlatformService.retrieveStaff(staffId);
         if (settings.isTemplate()) {
-            final Collection<OfficeData> allowedOffices = officeReadPlatformService.retrieveAllOfficesForDropdown();
+            final SearchParameters searchParameters = SearchParameters.forOffices("name", "ASC");
+            final Collection<OfficeData> allowedOffices = officeReadPlatformService.retrieveAllOffices(false, searchParameters);
```
Comment (mohitsinha) [inline fineract-provider/src/main/java/org/apache/fineract/organisation/staff/api/StaffApiResource.java:148]: Please update an existing testcase for this.
Or write a new one.

**Comment by adamsaghy** [comment]
@faheem205 Please follow the PR naming and commit naming conventions:
- Add FINERACT-XXXX story number into the PR title: (FINERACT-XXXX: <short description of PR>)
- also into the commit message (FINERACT-XXXX: <short description of commit>)

**Comment by vidakovic** [comment]
@faheem205 ... this doesn't seem to be related to a Fineract Jira ticket... could you please provide one (if it doesn't exist in Apache Jira) and change the PR title ("FINERACT-1234: ..."). Please make sure to use the exact pattern that (almost) everyone else uses. Thanks.

**Comment by vidakovic** [comment]
Doesn't seem to be worked on anymore. Closing.

**Pull request closed without merging**